### PR TITLE
i18n(zh-cn): Translate deno.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate deno.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/deno.mdx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+现在，你可以使用这个命令在本地使用 Deno 预览你的 Astro 站点。
+```bash
+npm run preview
+```
+
+## 使用
+
+{/* 翻译 */}
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/deno.mdx:79]:
```suggestion
```

File: src/content/docs/zh-cn/guides/integrations-guide/deno.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+在[执行构建](/zh-cn/guides/deploy/#在本地构建站点)之后，将会有一个 `dist/server/entry.mjs` 模块。你可以通过在你的 Deno 应用中导入这个模块来启动一个服务器：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/deno.mdx:80]:
```suggestion
在 [执行构建](/zh-cn/guides/deploy/#在本地构建站点) 之后，将会有一个 `dist/server/entry.mjs` 模块。你可以通过在你的 Deno 应用中导入这个模块来启动一个服务器：
```

**Review by 100gle — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**